### PR TITLE
chore: disable and assert disablility of telemetry on CI

### PR DESCRIPTION
**Pull request by sbalaji1192**
> Pull Request Template
>
> Use this template to quickly create a well written pull request. Delete all quotes before creating the pull request.

## Description

> Please include a summary of the changes and which issue has been fixed. Please also include relevant motivation
> and context. List any dependencies that are required for this change.

Fixes # (issue)

> if no issue exists, please create an issue and ask the maintainers about this first

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions, so we can reproduce.
> Please also list any relevant details for your test configuration.

- Test A
- Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>

    // Code coverage diff between base branch:release and head branch: chore/CI-issue-telemetry 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.71 **(-0.23)** | 40.83 **(-0.84)** | 36.21 **(0)** | 56.74 **(-0.25)**</details>

### CODE REVIEW

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Auth/Analytics_spec.js [hunk 1]
```diff
@@ -4,12 +4,14 @@ let appId;
 
 describe("Checks for analytics initialization", function() {
   it("Should check analytics is not initialised when enableTelemtry is false", function() {
-    cy.intercept("GET", "/api/v1/users/me", {
-      body: { responseMeta: { status: 200, success: true }, data: User },
-    }).as("getUsersWithoutTelemetry");
     cy.visit("/applications");
     cy.reload();
-    cy.wait("@getUsersWithoutTelemetry");
+    cy.wait(3000);
```
Comment (mohanarpit) [inline app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Auth/Analytics_spec.js:9]: Why are we waiting for 3 secs here? We are already waiting for `@getUser` right?

Comment (sbalaji1192) [reply app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Auth/Analytics_spec.js:9]: it was failing intermittently. due to the reload above. so had put a wait time.

Comment (rishabhsaxena) [reply app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Auth/Analytics_spec.js:9]: hey QQ: what's the use-case for triggering a reload? 
Also, we could add another check by adding a spy on the `initializeAnalyticsAndTrackers` method and verifying it wasn't called

**Review by mohanarpit — APPROVED** [review]

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Auth/Analytics_spec.js (hunk 1 above)
Comment (sbalaji1192) [reply app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Auth/Analytics_spec.js:9]: We register the analytics on load. so we want to explicitly reload and make sure that analytics objects are not getting created. initializeAnalyticsAndTrackers will create the objects on the window. so I think checking if those objects are present or not is enough. no need for a redundant check.

**Pull request merged**